.github/workflows/lint_pr.yml

```diff
@@ -0,0 +1,17 @@
+name: "Lint PR"
+
+on:
+  pull_request_target:
+    types:
+      - opened
+      - edited
+      - synchronize
+
+jobs:
+  main:
+    name: Validate PR title
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: amannn/action-semantic-pull-request@v5
```

Comment:
I'm reluctant to use third party github actions, for security reasons: https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#using-third-party-actions
Is there no way we can achieve this without a third-party action?

Reply:
Agreed. Sadly I could not find any more "official" way of doing things.  Basically we just need something that can take the PR title and evaluate it with [conventional-changelog](https://github.com/conventional-changelog/conventional-changelog) (which is what semantic-release uses under the hood.  Unfortunately, I could not find any official actions supported by `semantic-release` or `conventional-changelog`.

The code is pretty simple, though, so I guess if we were really interested in this functionality, we could go the fork/roll-our-own approach....